os-eap7-sso-adapters/configure.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+SCRIPT_DIR=$(dirname $0)
+ADDED_DIR=${SCRIPT_DIR}/added
+SOURCES_DIR="/tmp/artifacts"
+
+unzip -o $SOURCES_DIR/rh-sso-7.4.5-eap7-adapter.zip -d $JBOSS_HOME
+unzip -o $SOURCES_DIR/rh-sso-7.4.5-saml-eap7-adapter.zip -d $JBOSS_HOME
```

Comment:
Optional suggestion can we have double qoutes around the `$SOURCES_DIR` so it won't have any problem in case o any special chars or spaces in the var?

Reply:
sure.